Userland/Services/SystemServer/ClientConnection.cpp

```diff
@@ -59,8 +59,12 @@ OwnPtr<Messages::ServiceManagementServer::ServiceStatusResponse> ClientConnectio
 {
     auto service = ServiceManagement::the().find_service_by_name(message.service_name());
     dbgln("{}", service);
-    if (service)
-        return make<Messages::ServiceManagementServer::ServiceStatusResponse>(String::formatted("{}", service->is_enabled() ? "enabled" : "disabled"));
+    if (service) {
+        // FIXME: Handle multi-instance processes
+        auto enabled = service->is_enabled() ? "enabled" : "disabled";
+        auto running = service->is_running() ? "running" : "";
+        return make<Messages::ServiceManagementServer::ServiceStatusResponse>(String::formatted("{} {}", enabled, running));
```

Comment:
I'm not sure if I like formatting these into a string server-side. Perhaps it could return typed info (here, two booleans), and format them in the `service` utility?

Reply:
Ah, I see the next commit changes this to JSON. That's better, but perhaps explicit types at the IPC level would be better still.